packages/ui/src/components/Donut.js

```diff
@@ -156,7 +155,6 @@ export default class Donut extends Component {
             {this.renderSprinkles(frostingInnerRadius, frostingOuterRadius)}
           </g>
         </svg>
-        <Rating {...this.props} />
```

Comment:
`Donut` now doesn't have a background or include ratings. All the old components now include the `Rating` component directly.